TASK-4/EmployeeReportGenerator.cs

```diff
@@ -0,0 +1,35 @@
+namespace Persons;
+
+public class EmployeeReportGenerator : IReportGenerator
+{
+    // отсортировывает всех работников по компании и по убыванию зп.
+    // ***Вывод : UserId || Company Name || Users Full Name || Salary***
+
+    public IEnumerable<Employee> ListOfEmployees;
+
+    public EmployeeReportGenerator(IEnumerable<Employee> pear)
+    {
+        ListOfEmployees = pear;
+    }
+
+    public IEnumerable<Employee> DescendingSort()
+    {
+        return ListOfEmployees.OrderByDescending(e => String.Format("{0,70}{1,10}", e.CompanyName, e.Salary));
```

Comment:
Your sort doesn't work correctly, it doesn't sort at all (for companies, for salary)

Reply:
Fixed it.